packages/ui/src/domainModules/locality/components/LocalityManager/item/BaseForm/unitSpecs/index.js

```diff
@@ -1,5 +1,7 @@
+import localityRecordHistoryEvents from './localityRecordHistoryEvents'
 import place from './place'
 
 export default {
+  localityRecordHistoryEvents,
```

Comment:
one point of using this structure with sections and units is to avoid unnecessary duplicated code. the recordHistoryEvents unit looks the same in most places, maybe the model is different then it can be made as a factory, but if it is exactly the same we should import the unit from the form-module and just use it here. not copy paste everywhere